frontend/player/package.json

```diff
@@ -22,15 +23,15 @@
 		"dist"
 	],
 	"devDependencies": {
-		"@typescript-eslint/eslint-plugin": "^5.62.0",
-		"@typescript-eslint/parser": "^5.62.0",
+		"@typescript-eslint/eslint-plugin": "^6.0.0",
+		"@typescript-eslint/parser": "^6.0.0",
 		"astring": "^1.8.6",
-		"eslint": "^8.44.0",
+		"eslint": "^8.45.0",
 		"eslint-config-prettier": "^8.8.0",
-		"prettier": "^2.8.8",
-		"rimraf": "^3.0.2",
+		"prettier": "^3.0.0",
+		"rimraf": "^5.0.1",
 		"typescript": "^5.1.6",
-		"vite": "^4.4.3",
-		"vite-plugin-dts": "^2.3.0"
+		"vite": "^4.4.4",
+		"vite-plugin-dts": "^3.3.0"
```

Comment:
I usually don't trust that dependencies will follow semver and let everything be fixed.

Reply:
Most do, but if they dont our UTs will usually catch the issues. 

I added a clause in the pr description abt the changes in gql.ts

Reply:
But also, this is a precursor to the website redesign. We are pushing it forward because CI is failing due to website builds.